imops/morphology.py

```diff
@@ -7,26 +7,26 @@
 from scipy.ndimage import distance_transform_edt as scipy_distance_transform_edt, generate_binary_structure
 from scipy.ndimage._nd_image import euclidean_feature_transform
 from scipy.spatial import ConvexHull, QhullError
+from skimage._shared.utils import warn as skimage_warn
```

Comment:
why do you need skimage_warn? isn't the regular `warnings` module enough?

Reply:
Just copied code from skimage, idk why they use custom warn)

Reply:
let's get rid of imports from protected submodules where we can. this is one clearly can be replaced by `warnings`

Reply:
Fair, fixed that